packages/components/date-picker/src/date-picker-com/panel-date-pick.vue

```diff
@@ -253,6 +253,8 @@ const innerDate = ref(dayjs().locale(lang.value))
 
 const isChangeToNow = ref(false)
 
+const isShortcut = ref(false)
```

Comment:
```suggestion
let isShortcut = false
```
I think here just need to declare a normal variable on the line.

Reply:
Done.